sway/commands/focus.c

```diff
@@ -377,6 +377,10 @@ struct cmd_results *cmd_focus(int argc, char **argv) {
 		if (container_is_scratchpad_hidden_or_child(container)) {
 			root_scratchpad_show(container);
 		}
+		struct sway_container *obstructing = container_obstructing_fullscreen_container(container);
```

Comment:
Probably best to add a comment here for why `seat_set_focus_container` is insufficient, since this is the only special case.